src/ContextTypes/Recipe.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace JsonLd\ContextTypes;
+
+class Recipe extends CreativeWork
+{
+    /**
+     * Property structure
+     * reference: https://schema.org/Recipe (alphabetical order)
+     *
+     * @var array
+     */
+    protected $structure = [
+        'name' => null,
```

Comment:
you must not repeat an entry that already exists in parents (Creative Work or Thing).